docs/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Vercel
+meta_desc: Provides an overview of the Vercel Provider for Pulumi.
+layout: overview
```

Comment:
Value for `layout` must be `package`.

Reply:
This was taken from the example you sent..